src/solver/expressions/include/antares/solver/expressions/visitors/AstGraphVisitor.h

```diff
@@ -0,0 +1,82 @@
+/*
+** Copyright 2007-2024, RTE (https://www.rte-france.com)
+** See AUTHORS.txt
+** SPDX-License-Identifier: MPL-2.0
+** This file is part of Antares-Simulator,
+** Adequacy and Performance assessment for interconnected energy networks.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the Mozilla Public Licence 2.0 as published by
+** the Mozilla Foundation, either version 2 of the License, or
+** (at your option) any later version.
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** Mozilla Public Licence 2.0 for more details.
+**
+** You should have received a copy of the Mozilla Public Licence 2.0
+** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
+*/
+#pragma once
+
+#include <ostream>
+#include <utility>
+
+#include "antares/solver/expressions/visitors/NodeVisitor.h"
+
+namespace Antares::Solver::Visitors
+{
+
+struct BoxStyle
+{
+    std::string label;
+    std::string color = "azure";
+    std::string shape = "box";
+    std::string style = "rounded";
+};
+
+class AstGraphVisitor: public NodeVisitor<void, std::ostream&>
```

Comment:
This name isn't very explicit, why not `DOTStreamVisitor` ?

Reply:
Do you really think that DOTStream is more explicit than AstGraph?
I would say AstGraphDotStyle

Reply:
Yes, but I think DOT must be capitalized entirely (on official docs it's DOT)

Reply:
ok